code_schemes/s05e01_reasons.json

```diff
@@ -120,6 +120,24 @@
       "NumericValue": -100040,
       "StringValue": "opt_in",
       "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-SC-a3a065bc",
+      "CodeType": "Meta",
+      "MetaCode": "similar_content",
+      "DisplayText": "similar content",
```

Comment:
Could you check with the RAs, I think this is risking getting confusing have a long list where some of the codes are substantive, others are control, others are meta. Consider

- Sorting the codes in the drop down by category
and/or
- Prefixing the display text with Meta:

Reply:
These are already sorted by category: Normal, then Control, then Meta:
<img width="278" alt="image" src="https://user-images.githubusercontent.com/7963608/69863157-09f91500-1294-11ea-88ce-e03f375ac27a.png">

Reply:
The codes are already sorted by category. 2 of 3 RAs have got back to me saying there's no problem. I think we merge as-is, we can easily change the display text in future should we need to.